### PR TITLE
changelog: fix formatting

### PR DESCRIPTION
**Pull request by psergee**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [2.5.2] - 2024-11-07
 
-Update luarocks and libraries version.
+- Update luarocks and libraries version.
```
Comment (oleg-jukovec) [inline CHANGELOG.md:20]: I disagree, this is just a release overview.

File: CHANGELOG.md [hunk 2]
```diff
@@ -32,11 +32,11 @@ Update luarocks and libraries version.
...
 
-Added a number of subcommands for replicaset roles,
-improved understanding of the version number to install,
-enhanced status display, and added a flag to disable interactivity when
+- Added a number of subcommands for replicaset roles.
+- Improved understanding of the version number to install,
+- Enhanced status display, and added a flag to disable interactivity when
 the application is stopped.
-Implemented a number of fixes to improve stability.
+- Implemented a number of fixes to improve stability.
```
Comment (oleg-jukovec) [inline CHANGELOG.md:39]: The same.

File: CHANGELOG.md (hunk 1 above)
Comment (psergee) [reply CHANGELOG.md:20]: What is "release overview message" and why is it in the change log?

File: CHANGELOG.md (hunk 2 above)
Comment (psergee) [reply CHANGELOG.md:39]: The same.

**Pull request closed without merging**